node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/util/ConfigServerHttpRequestExecutor.java

```diff
@@ -110,6 +115,15 @@ public static ConfigServerHttpRequestExecutor create(Collection<URI> configServe
         Exception lastException = null;
         for (int loopRetry = 0; loopRetry < MAX_LOOPS; loopRetry++) {
             for (URI configServer : configServerHosts) {
+                if (lastException != null) {
```

Comment:
Do we need to timeout between trying different config servers? Isn't the problem that we get to second iteration and retry against the same cfg. server again? If so, we could have this sleep at the end of the first loop with maybe a longer timeout, and this wouldn't affect the upgrade rollout speed.

Reply:
Discussed with Harald. Let's just remove the retrying against the same config server and rely on the next tick to retry.